src/mlpack/methods/decision_tree/decision_tree.hpp

```diff
@@ -15,19 +15,19 @@
 
 #include <mlpack/core.hpp>
 
-#include "gini_gain.hpp"
-#include "information_gain.hpp"
-#include "mad_gain.hpp"
-#include "mse_gain.hpp"
+#include "gain_functions/gini_gain.hpp"
+#include "gain_functions/information_gain.hpp"
+#include "gain_functions/mad_gain.hpp"
+#include "gain_functions/mse_gain.hpp"
+#include "gain_functions/sse_gain.hpp"
```

Comment:
Many other subdirectories have files like `gain_functions/gain_functions.hpp` that simply include all the gain functions---do you want to do that here too?  It's just a convenience.

Reply:
Oh yeah, I'll do that now.